src/Windows/Avalonia.Win32/WindowImpl.cs

```diff
@@ -731,8 +731,9 @@ private void CreateWindow()
                 RegisterTouchWindow(_hwnd, 0);
             }
 
-            if (ShCoreAvailable)
-            {
+            if (ShCoreAvailable && (Win32Platform.WindowsVersion.Major > 6 || 
```

Comment:
Define Windows8 as a constant. So if statements can read if(ShCoreAvailable && Win32Platform.WindowsVersion > Windows8

Reply:
I've added it in the respective PlatformConstants.cs file in the same assembly.